src/pages/filter/dtree/dtree.page.tsx

```diff
@@ -40,14 +41,17 @@ export const DtreePage = observer((): ReactElement => {
       })
     }
 
-    dtreeName ? filterDtreesStore.setActiveDtree(dtreeName) : initAsync()
+    if (filterDtreesStore.activeDtree && !dtreeName) {
+      pushQueryParams({ dtree: filterDtreesStore.activeDtree })
+      return
+    } else if (dtreeName) {
```

Comment:
Remove else here, it is useless

Reply:
Do you mean remove  `else { initAsync() } `or what?